db/schema.rb

```diff
@@ -10,14 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_03_18_112417) do
+ActiveRecord::Schema.define(version: 2022_03_24_162356) do
```

Comment:
I think this is the only bit of the diff on this file we should commit. Can you back the other bits out?